motion_adaption:
  # name of this adaption
  adaption_name: "robosem_head_adaption"
  # adaption type (no_adaption, trans_rot_adaption)
  adaption_type: "trans_rot_adaption"
  # reference frame for the input motion
  input_ref_frame: "input_ref_link"
  # parent frame name for setting the input reference frame
  input_ref_dep_parent: "camera_link"
  # child frame name for setting the input reference frame
  input_ref_dep_child: "torso_1"
  # correction needed to align the input reference frame's orientation with the target reference frame's
  input__ref_correction: {0.0, 0.0, 0.0}
  # endpoint, which motion is retargeted 
  input_endpt: {"test_head_link"}
  # reference frame for the retargeted motion
  output_ref_frame: "base_link"
  # endpoint for the retarget motion
  output_endpt: {"head_goal"}
  # applied rotational correction for the input motion
  input_correction: {0.0, 0.0, 0.0}
